test/index.test.js

```diff
@@ -4,6 +4,7 @@ var persistState = require('../')
 
 afterEach(function () {
   localStorage.clear()
+  jest.restoreAllMocks()
```

Comment:
The `JSON.stringify` spy on line 70 would prevent usage of `JSON.stringify` in all subsequent tests.

Reply:
:+1: